src/frontend/js/order/orderApi.js

```diff
@@ -1,4 +1,29 @@
+/* globals API_URL */
 import { fetchEntitiesFactory, fetchEntityFactory } from '../app/entities/fetchEntities';
 
 export const fetchOrders = fetchEntitiesFactory('orders');
 export const fetchOrder = fetchEntityFactory('orders');
+
+export const fetchNewOrder = (products, jwt) => fetch(`${API_URL}/orders`, {
```

Comment:
The `fetchEntityFactory` is only used to load an entity. When we'll have several post on a single item, it will be worth to consider upgrading it